components/AnswerSection/index.tsx

```diff
@@ -21,6 +21,9 @@ import {
   Tts,
```

Comment:
In General I think this file needs to be split up into multiple separate components